src/proto/Cosmos.proto

```diff
@@ -28,6 +28,23 @@ message Message {
         string type_prefix = 4;
     }
 
+    // cosmos-sdk/MsgTransfer, IBC transfer
+    message Transfer {
+        // IBC port, e.g. "transfer"
+        string source_port = 1;
+        // IBC connection channel, e.g. "channel-141"
+        string source_channel = 2;
```

Comment:
put them in a `Source` message?

Reply:
Let's keep analogue to the internal proto structure.